geospatial/dted.ksy

```diff
@@ -34,19 +34,19 @@ types:
       - id: recognition_sentinel
         contents: UHL
         doc: |
-          Recognition sentinel
+          Recognition sentinel ("UHL")
```

Comment:
It is unneeded to put the strings into docs. The `ksy` file is docs itself.

Reply:
Ok